Accept newer releases 4.1 and 4.2 also

kraj opened the pull request:
These releases of upstream yocto work fine with kirkstone branch. Therefore we can support them on single kirkstone branch without affecting the kirkstone LTS release which is officially supported by variscite.

kraj committed:
layer.conf: Add langdate ( 4.1 ) to compatible layers

Signed-off-by: Khem Raj <raj.khem@gmail.com>

kraj committed:
variscite: Use a wildchar for preferred gstreamer

master has 1.20.3 unlike kirkstone which uses 1.20.0

Signed-off-by: Khem Raj <raj.khem@gmail.com>

kraj committed:
bluealsa: Add missing SRC_URI entry for bluealsa.service

Signed-off-by: Khem Raj <raj.khem@gmail.com>

kraj committed:
layer.conf: Update LAYERSERIES_COMPAT for mickledore

oe-core switched to mickedore [1]

[1] https://git.openembedded.org/openembedded-core/commit/?id=57239d66b933c4313cf331d35d13ec2d0661c38f

Signed-off-by: Khem Raj <raj.khem@gmail.com>

nsdrude reviewed (commented):
@kraj Regarding https://github.com/varigit/meta-variscite-bsp/pull/12/commits/d68a039a8b0b2d1f96b0471412dbfb7e5c4c749b, most of these SoMs have already been updated to 5.15 (or will be in the near future). See https://variwiki.com/

Also, meta-arm and therefore arm-ffa-tee is not included by our manifest, so this bbappend will cause an error.

kraj commented:
> @kraj Regarding [d68a039](https://github.com/varigit/meta-variscite-bsp/commit/d68a039a8b0b2d1f96b0471412dbfb7e5c4c749b), most of these SoMs have already been updated to 5.15 (or will be in the near future). See https://variwiki.com/
> 
> Also, meta-arm and therefore arm-ffa-tee is not included by our manifest, so this bbappend will cause an error.

I will remove arm-ffa-tee patch from the pull. Do we have imx8qm-var-som working with 5.15 now ?

nsdrude commented:
@kraj Yes, Yocto Kirkstone with 5.15 is available for the imx8qm-var-som: https://variwiki.com/index.php?title=VAR-SOM-MX8

kraj committed:
imx-atf: Fix build with latest binutils and compiler

These patches are brought from meta-freesale master branch since SRC_URI
is overridden by this bbappend, we still need to apply these two patches
to make it work with newer gcc/binutils from newer yocto releases

Signed-off-by: Khem Raj <raj.khem@gmail.com>

kraj committed:
imx-gst1.0-plugin: Do not set PREFERRED_VERSION

Its redundant, because there is only one version of recipe in
meta-freescale, this makes the layer portable for newer yocto releases
without impacting kirktone

Signed-off-by: Khem Raj <raj.khem@gmail.com>
